translations/remote-schemas.kannada.md

```diff
@@ -0,0 +1,83 @@
+# ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು
+
+ನೀವು ಎಲ್ಲಾ ಪ್ರಕಾರದ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಅನ್ನು ಒಂದೇ ಸ್ಥಳದಿಂದ ಪ್ರಶ್ನಿಸಲು ಬಯಸಿದರೆ ನಂತರ ದೂರಸ್ಥ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸ್ಕೀಮಾವನ್ನು ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಂಜಿನ್‌ನ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್ ಆಧಾರಿತ ಸ್ಕೀಮಾದೊಂದಿಗೆ ವಿಲೀನಗೊಳಿಸಿ. ರಿಮೋಟ್ ಸ್ಕೀಮಾಗಳು ಕೆಲವು ಉದಾಹರಣೆಗಳಿಗೆ ಸೂಕ್ತವಾಗಿವೆ:
```

Comment:
This sentence requires improved translation to provide the same understanding as the English version.
 
 This preceding sentence contains no information on (Hasura has the ability to merge remote GraphQL schemas and provide a unified GraphQL API.)

Reply:
Thank you for your review comments @harishbala1o 
I have incorporated the necessary changes. Do let me know if there is anything else that requires rephrasing.